src/intervals/arithmetic.jl

```diff
@@ -162,8 +162,8 @@ function inv(a::Interval{T}) where T<:Real
     isempty(a) && return emptyinterval(a)
 
     if zero(T) ∈ a
-        a.lo < zero(T) == a.hi && return @round(-Inf, inv(a.lo))
-        a.lo == zero(T) < a.hi && return @round(inv(a.hi), Inf)
+        a.lo < zero(T) == a.hi && return @round(T(-Inf), inv(a.lo))
+        a.lo == zero(T) < a.hi && return @round(inv(a.hi), T(Inf))
```

Comment:
Since `T(Inf)` is used quite a bit, it may be worth to define `infinity(T)` to work the same way `zero(T)` does.

Reply:
I fully agree (though maybe in another PR).